examples/apps/attributable-map/src/modelContainerRuntimeFactoryWithAttribution.ts

```diff
@@ -12,12 +12,14 @@ import {
 	IRuntimeFactory,
 } from "@fluidframework/container-definitions/internal";
 import {
+	// eslint-disable-next-line import/no-deprecated -- ContainerRuntime class to be moved to internal scope
 	ContainerRuntime,
 	IContainerRuntimeOptions,
 } from "@fluidframework/container-runtime/internal";
 import { IContainerRuntime } from "@fluidframework/container-runtime-definitions/internal";
 import { NamedFluidDataStoreRegistryEntries } from "@fluidframework/runtime-definitions/internal";
 
+// eslint-disable-next-line import/no-deprecated
 const containerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
 
```

Comment:
Examples should not be using /internal exports. (We need to fix the linting config so that it allows /beta, /alpha, and /legacy but rejects /internal and then fix them up.)
What is the correction for this usage?

Reply:
Wow luckily I can just remove the argument since it's the default value.  I was afraid the whole mixin was implemented in an Experimental package, which would have been a pain to deal with.  (I still think we can drop that pattern but don't want it coupled with this moment)